app/views/reports/_reports_table.html.erb

```diff
@@ -20,15 +22,18 @@
               <td><%= key %></td>
             <% end %>
             <% @date_headers.each do |date| %>
-              <td><%= record[date] %></td>
+              <td class="text-center <%= record[date] == 'Yes' ? 'bg-success' : (record[date] == 'No'? 'bg-danger' : '' ) %>
+              ">
```

Comment:
I like different colors for yes and no answers. But I would change the colors of the 'Yes' and 'No' words, not the background color. I think that would be too much if every cell on the page had data and was red or green.